install.sh

```diff
@@ -154,6 +153,45 @@ __atuin_install_cargo(){
 	cargo install atuin
 }
 
+__atuin_install_binary() {
+    local target
+    local arch=$(uname -sm)
+    local bin_dir=/usr/local/bin
+
+    # todo: Can make it an optional argument to install script as well,
+    # which could support:
+    # bash ~/.local/bin/ <(curl --proto '=https' --tlsv1.2 -sSf https://setup.atuin.sh)
+    # local bin_dir=${1:-/usr/local/bin}
+
+    case $arch in
+    "Linux x86_64")
+        target="x86_64-unknown-linux-musl"
+        ;;
+    "Linux aarch64")
+        target="aarch64-unknown-linux-gnu"
+        ;;
+    "Darwin x86_64")
+        target="x86_64-apple-darwin"
+        ;;
+    "Darwin arm64")
+        target="aarch64-apple-darwin"
+        ;;
+    *)
+        __atuin_install_unsupported
+        return
+        ;;
+    esac
+
+    local latest_version=$(curl -s https://api.github.com/repos/atuinsh/atuin/releases/latest | grep tag_name  | grep -Eo "v[0-9]+\.[0-9]+\.[0-9]+")
+    local archive_name=atuin-$latest_version-$target
+    local binary_url="https://github.com/atuinsh/atuin/releases/download/v17.0.1/$archive_name.tar.gz"
+    download_dir=$(mktemp -d)
+    curl -s -L $binary_url | tar -xz -C $download_dir -f -
+    sudo mkdir -p $bin_dir
```

Comment:
I think it would be good to 

1) Let the user know what's going on (eg tell them we couldn't find anything + will install a binary from releases)
2) Tell them where it's being installed, and ask for y/n to go ahead

I don't think we should go putting files in the users filesystem without getting their confirmation that is OK, especially when they'll be getting a `sudo` prompt without explanation

Reply:
True. I think I can add logging here, with a prompt.